website/integrations/services/espoCRM/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Integrate with espoCRM
+sidebar_label: espoCRM
+---
+
+# espoCRM
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is espoCRM?
+
+> EspoCRM is a web application that allows users to see, enter and evaluate all your company relationships regardless of the type. People, companies, projects or opportunities — all in an easy and intuitive interface.
+>
+> -- https://www.espocrm.com/
+
+:::warning
+This guide does _not_ cover Team Mapping. Please refer to espoCRM's [documentation](https://docs.espocrm.com/administration/oidc/#team-mapping).
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `crm.company` is the FQDN of the espoCRM install. 
+-   `authentik.company` is the FQDN of the authentik install.
+- `_SLUG_` is the slug you choose upon application create in authentik.
+
+Create an application in authentik and note the slug you choose, as this will be used later (`_SLUG_`).
+In the Admin Interface, go to **Applications** -> **Providers**. Create a **OAuth2/OpenID** provider with the following parameters:
+
```

Comment:
You can now use the wizard to create both an application and provider at the same time... I'd suggest you rewrite lines 28 and 29 to reflect that. Here's a suggestion:
+++++++++++++++++
Log into authentik as an admin, and navigate to **Applications --> Applications**.

Click **Create with Wizard**. (Alternatively, use our legacy process and click **Create**. The legacy process requires that the application and its authentication provider be configured separately.)

In the New application wizard, define the application details, and then define the provider with the following parameters:

Reply:
Great suggestion, thank you for reviewing!! I will work on this shortly. 

Also, I recently discovered batch commits for suggestions. I think I might have undid one of my previous commits (from one of your suggestions). I can't figure out how to fix it/apply your suggestion again. 

sorry in advance if it messed anything up in the background

Reply:
I just added your suggestions!